Latest/Interface/Update Table View/Controller/UpdateTableViewController.swift

```diff
@@ -129,6 +129,16 @@ class UpdateTableViewController: NSViewController, NSMenuItemValidation, NSTable
 		
         IconCache.shared.icon(for: app) { (image) in
             cell.imageView?.image = image
+            
+            // Tint the icon if the app is not supported
+            let supported: Bool
+            if let app = cell.app {
+                supported = type(of: app).supported
+            } else {
+                supported = false
+            }
+            
+            cell.imageView?.alphaValue = (supported ? 1 : 0.5)
```

Comment:
The app is already unwrapped in the outer scope. You can simply put the following line here:

Reply:
Ok. I'm more of a code noob.